_includes/configuration/iframe-aria-text.md

```diff
@@ -0,0 +1,18 @@
+## `iframe_aria_text`
+
+{{ site.requires_5_9v }}
+
+This option allows setting the editor iframe title attribute. It is intended for use with {{site.productname}}'s [classic mode]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) only, as the editable area is sandboxed within an iframe.
```

Comment:
```suggestion
This option is used to customize the `title` attribute on the {{site.productname}} `iframe` element. For example:
 
~~~html
<iframe title="Rich Text Area. Press ALT-0 for help."></iframe>
~~~

This `title` attribute is read by screen-readers to help users identify the editor. This option only applies to {{site.productname}} [classic (iframe) mode]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/). This option has no effect on [`inline` editors]({{site.baseurl}}/configure/editor-appearance/#inline).
```
Note: The `~` block is just different syntax for codeblocks. I can't use backticks without breaking the suggestion

Reply:
Hmm I would use a different string in the example as that one is the default value.

The last paragraph I'd do like this to avoid repeating "this":

The `title` attribute is read by screen-readers to help users identify the editor. This option only applies to {{site.productname}} [classic (iframe) mode]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) and has no effect on [`inline` editors]({{site.baseurl}}/configure/editor-appearance/#inline).

What do you think?

Reply:
I've updated the suggestion, however I've also removed the link to classic editor mode because it's mid-sentence and fairly redundant given the second half of the sentence.